src/main/java/jp/co/future/uroborosql/event/ExecutionConsumer.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2017-present, Future Corporation
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+package jp.co.future.uroborosql.event;
+
+import java.sql.SQLException;
+
+/**
+ * SQLやProcedureを実行する際に利用するConsumer.<br>
+ * SQLExceptionをスローするため、{@link java.util.function.Consumer} とは別に作成する.
+ *
+ * @param <T> the type of the input to the operation
+ */
+@FunctionalInterface
+public interface ExecutionConsumer<T> {
+	/**
+	 * 指定された引数でこのオペレーションを実行します.
+	 *
+	 * @param t 引数
+	 * @throws SQLException SQL例外発生時
+	 */
+	void accept(T t) throws SQLException;
```

Comment:
Shouldn't it be UroborosqlSQLException instead of a checked exception?

Reply:
It was assumed that the Consumer implementation could just throw SQLException as is, or wrap it in UroborosqlRuntimeException and throw it if necessary.

Reply:
OK. Let's leave it like this.